### ISSUE TITLE
Quests and Adventure Hooks

### ISSUE THREAD

**Issue by jakethedev** [opendnd/opendnd#3]
Hi there - I've been working here and there on github.com/jakethedev/tavernbot (was jakerunsdnd but that got old lol), and I've got some work done in JS for generating quests and hooks. Would that be useful for the opendnd project? It's stashed in the Dungeonary folder, which is published to npm as a standalone lib

I'm just about in a 'done' state with the bot and it would be cool for some of the late game features like quests to get used in more places, so I figured I'd ask and see if that would be cool before planning on forking and making a PR :) It'll be a bit before it's all done and ready

**Comment by drewry** [comment]
@jakethedev absolutely that would be a great addition to OpenDnD! Quests are currently on the OpenDnD roadmap as the [https://github.com/opendnd/api/tree/master/quests](Questae tool).  The idea is that OpenDnD is built up of smaller projects like [https://github.com/opendnd/avataria](Avataria) for avatars, [https://github.com/opendnd/personae](Personae) for generating characters, etc.

I went ahead and created the Questae repo. Would you want to PR into that? Then we can reference it from the main OpenDnD package. This way the quest tool can reference the existing NPM package and pull from those features and wrap into the existing OpenDnD tool system.

Please let me know if that makes sense, I'm excited to see this addition :)

**Comment by jakethedev** [comment]
Yeah, that sounds awesome! Glad you're on the discord where the bot lives too, that'll make chatting about this way more straightforward 😄 

Per the discord chat, and just for logging it here, the quest generator's at about 90% functionality and 65% usable - it just needs more content instead of certain chunks of placeholders, and the settings outside of high fantasy need fleshed out, but could be easy to finish with a handful of hours of work, so it'll be a bit but it'll be fairly extensive, and should be able to support full adventure hooks for several settings - plus possible quest stub generation
